cpp/ql/src/semmle/code/cpp/models/interfaces/Deallocation.qll

```diff
@@ -30,3 +30,11 @@ abstract class DeallocationExpr extends Expr {
    */
   Expr getFreedExpr() { none() }
 }
+
+/**
+ * An `operator delete` or `operator delete[]` function that may be associated
+ * with `delete` or `delete[]` expressions.  Note that `delete` and `delete[]`
+ * are not function calls, but these functions may also be called directly.
+ */
+abstract class OperatorDeleteDeallocationFunction extends DeallocationFunction {
```

Comment:
Same questions as for `OperatorNewAllocationFunction`

Reply:
Done as above.